### PR TITLE
cloud9: fix broken images in Quick Start in Safari

### PR DESCRIPTION
**Pull request by gsweetwood**
## Problem
The Quick Start webview will not render images in Safari.

## Solution
Enabling scripts for the webview should allow the images to render.
Set the default CSP.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: quickStartCloud9.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<head><meta http-equiv="Content-Security-Policy" content="default-src 'none'"></head>
```
Comment (justinmk3) [inline quickStartCloud9.html:1]: Why was this committed?

**Comment by justinmk3** [comment]
Testing this PR in safari + cloud9, when I run `AWS: View Quick Start`, now I see an error "Webview is violating CSP rules".

File: quickStartCloud9.html (hunk 1 above)
Comment (gsweetwood) [reply quickStartCloud9.html]: removed

**Review by justinmk3 — DISMISSED** [review]

**Review by justinmk3 — COMMENTED** [review]
Confirming working in Safari. But the tests are failing on some systems.

File: src/test/shared/extensionUtilities.test.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ describe('extensionUtilities', function () {
             const forcedWebview = webview as vscode.WebviewPanel
 
             const pathAsVsCodeResource = forcedWebview.webview.asWebviewUri(vscode.Uri.file(context.extensionPath))
-            assert.strictEqual(forcedWebview.webview.html, `${basetext}${pathAsVsCodeResource}`)
+            assert.strictEqual(forcedWebview.webview.html.includes(basetext + pathAsVsCodeResource.toString()), true)
```
Comment (justinmk3) [inline src/test/shared/extensionUtilities.test.ts:89]: why change this from string interpolation to "+" form?

Comment (gsweetwood) [reply src/test/shared/extensionUtilities.test.ts]: Fixed

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**